test/node/bson_test.js

```diff
@@ -1219,7 +1219,7 @@ describe('BSON', function() {
     expect(long instanceof Long).to.be.ok;
     expect(!(long instanceof Timestamp)).to.be.ok;
     expect(timestamp instanceof Timestamp).to.be.ok;
-    expect(!(timestamp instanceof Long)).to.be.ok;
+    // expect(!(timestamp instanceof Long)).to.be.ok;
```

Comment:
hm what's going on here?

Reply:
this test is "Should Correctly Serialize and Deserialize Long Integer and Timestamp as different types". but since this change just makes Timestamp a wrapper around Long, Timestamp will now be an instance of Long

Reply:
Yeah, but then shouldn't we just delete the expectation?  Are we anticipating that this will be true again some day?

Reply:
Or rather update the expectation to assert it _is_ a `Long`?

Reply:
ok good point I'll change it. deleting things entirely just feels more dramatic 😱